### PR TITLE
[14.0] Repo Updated from template

### PR DESCRIPTION
**Pull request by bosd**
Did a copier update according to https://github.com/OCA/oca-addons-repo-template

This should fix broken github actions

@OCA/l10n-netherlands-maintainers Can you merge this?

### CODE REVIEW

**Comment by bosd** [comment]
Should have changed this to github actions instead of travis CI.

**Comment by bosd** [comment]
@NL66278 It's ready

File: .pylintrc [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (NL66278) [inline .pylintrc]: Why the blank lines in this and next file?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      matrix:
+        include:
+          - container: ghcr.io/oca/oca-ci/py3.6-odoo14.0:latest
+            makepot: "true"
+            name: test with Odoo
+          - container: ghcr.io/oca/oca-ci/py3.6-ocb14.0:latest
+            name: test with OCB
+    services:
+      postgres:
+        image: postgres:9.6
```
Comment (NL66278) [inline .github/workflows/test.yml:45]: According to docs Odoo 14.0 needs Postgres 10.0 or higher.

Comment (bosd) [reply .github/workflows/test.yml:45]: Thanks for the thorough review. You have an good point here. 
https://www.odoo.com/documentation/14.0/administration/install/install.html#postgresql
>supported version: 10.0 and later

Just checked this with the actions config of 14.0-timesheet and 14.0-project repo.
They all run on postgres 9.6

This is something that should be changed in the repo template.
You might want to open a ticket there.

File: .pylintrc (hunk 1 above)
Comment (bosd) [reply .pylintrc]: I tried to get them out.
But this is something coming from the main template repo:
https://github.com/OCA/edi/pull/686/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9

**Comment by bosd** [comment]
@NL66278 Can we proceed to merge these repo update PR's?

**Comment by NL66278** [comment]
@bosd There are conflicts. Is this PR still needed? If so, can you resolve the conflicts?

**Comment by bosd** [comment]
@NL66278 Yes, it is still needed. Conflicts are resolved.

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Pull request merged**